File: home/dot_local/share/lazyvim/lib/versions.mjs
import fs from "node:fs";
import path from "node:path";

const targetHome =
  process.env.CHEZMOI_DESTDIR || process.env.HOME || process.env.USERPROFILE;
if (!targetHome)
  throw new Error("Unable to determine the target home directory");

export const versions = Object.freeze({
  fdMajor: "10",
  fzf: "0.74.2",
  go: "1.27.0",
  lazygit: "0.63.1",
  neovim: "0.12.4",
  node: fs.readFileSync(path.join(targetHome, ".node-version"), "utf8").trim(),
  nvmWindows: "1.2.2",
  ripgrep: "15.2.0",
  treeSitter: "0.26.11",
});

File: home/dot_local/share/lazyvim/lua/lazyvim_capabilities/versions.lua
local paths = require("lazyvim_capabilities.paths")

return {
	fd_major = "10",
	fzf = "0.74.2",
	go = "1.27.0",
	lazygit = "0.63.1",
	neovim = "0.12.4",
	node = vim.trim(paths.read(paths.join(paths.home, ".node-version"))),
	nvm_windows = "1.2.2",
	ripgrep = "15.2.0",
	tree_sitter = "0.26.11",
}
